### PR TITLE
exim 4.97

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  Cannot read timezone file /usr/share/zoneinfo/UTC0 (from TZDIR/TZ)
  	'man tzset' may help.
  >>> exim_msgdate script built
  Can't locate File/FcntlLock.pm in @INC (you may need to install the File::FcntlLock module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.34.0 /usr/local/share/perl/5.34.0 /usr/lib/x86_64-linux-gnu/perl5/5.34 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl-base /usr/lib/x86_64-linux-gnu/perl/5.34 /usr/share/perl/5.34 /usr/local/lib/site_perl) at ./exim_id_update line 18.
  BEGIN failed--compilation aborted at ./exim_id_update line 18.
  make[1]: *** [Makefile:577: exim_id_update] Error 2
  make[1]: Leaving directory '/tmp/exim-20231104-3636-1q1bpoi/exim-4.97/build-Linux-x86_64'
```

File: Formula/e/exim.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Exim < Formula
   desc "Complete replacement for sendmail"
   homepage "https://exim.org"
-  url "https://ftp.exim.org/pub/exim/exim4/exim-4.96.2.tar.xz"
```
Comment (chenrui333) [inline Formula/e/exim.rb:4]: we probably need a mirror for it, the old artifact is gone when upstream makes a new release

Comment (cho-m) [reply Formula/e/exim.rb:4]: Looks like old tarballs get moved to `old/` directory, e.g. https://ftp.exim.org/pub/exim/exim4/old/exim-4.96.2.tar.xz

Comment (chenrui333) [reply Formula/e/exim.rb:4]: good catch!

**Comment by chenrui333** [comment]
only linux build failed now

**Review by krehel — APPROVED** [review]

**Pull request merged**